kura/test/org.eclipse.kura.nm.test/src/test/java/org/eclipse/kura/nm/NMDbusConnectorTest.java

```diff
@@ -0,0 +1,897 @@
+
+        when(mockedDevice1.getObjectPath()).thenReturn("/mock/device/" + interfaceName);
+
+        Generic mockedDevice1Generic = mock(Generic.class);
+        when(mockedDevice1Generic.getObjectPath()).thenReturn("/mock/device/" + interfaceName);
+
+        DBusPath mockedPath1 = mock(DBusPath.class);
+        when(mockedPath1.getPath()).thenReturn("/mock/device/" + interfaceName);
+
+        Map<String, Map<String, Variant<?>>> mockedDevice1ConnectionSetting = new HashMap<String, Map<String, Variant<?>>>();
+        mockedDevice1ConnectionSetting.put("connection",
+                Collections.singletonMap("uuid", new Variant<>("mock-uuid-123")));
+
+        Settings mockedDevice1Settings = mock(Settings.class);
+        when(mockedDevice1Settings.GetConnectionByUuid(eq("mock-uuid-123"))).thenReturn(mockedPath1);
+
+        GetAppliedConnectionTuple mockedDevice1ConnectionTouple = mock(GetAppliedConnectionTuple.class);
+        when(mockedDevice1ConnectionTouple.getConnection()).thenReturn(mockedDevice1ConnectionSetting);
+
+        when(mockedDevice1.GetAppliedConnection(eq(new UInt32(0)))).thenReturn(mockedDevice1ConnectionTouple);
+
+        Properties mockedProperties1 = mock(Properties.class);
+
+        if (interfaceName.equals("lo")) {
+            when(mockedProperties1.Get(any(), any())).thenReturn("loopback");
+        }
+        
+        if (interfaceName.toLowerCase().contains("wlan")) {
+        	
+        	Wireless wirelessDevice = mock(Wireless.class);
+        	when(wirelessDevice.getObjectPath()).thenReturn("/mock/device/" + interfaceName);
+        	
+        	ExitStatus exitStatus = mock(ExitStatus.class);
+        	when(exitStatus.getExitCode()).thenReturn(1);
+        	when(exitStatus.isSuccessful()).thenReturn(true);
+        	
+        	String commandOutput = "global\n"
+        			+ "country CA: DFS-FCC\n"
+        			+ "	(2402 - 2472 @ 40), (N/A, 30), (N/A)\n"
+        			+ "	(5150 - 5250 @ 80), (N/A, 23), (N/A), NO-OUTDOOR, AUTO-BW\n"
+        			+ "	(5250 - 5350 @ 80), (N/A, 24), (0 ms), DFS, AUTO-BW\n"
+        			+ "	(5470 - 5600 @ 80), (N/A, 24), (0 ms), DFS\n"
+        			+ "	(5650 - 5730 @ 80), (N/A, 24), (0 ms), DFS\n"
+        			+ "	(5735 - 5835 @ 80), (N/A, 30), (N/A)\n"
+        			+ "\n"
+        			+ "phy#0\n"
+        			+ "country 99: DFS-UNSET\n"
+        			+ "	(2402 - 2482 @ 40), (6, 20), (N/A)\n"
+        			+ "	(2474 - 2494 @ 20), (6, 20), (N/A)\n"
+        			+ "	(5140 - 5360 @ 160), (6, 20), (N/A)\n"
+        			+ "	(5460 - 5860 @ 160), (6, 20), (N/A)";
+        	
+        	String commandOutput2 = "Interface wlan0\n"
+        			+ "	ifindex 3\n"
+        			+ "	wdev 0x1\n"
+        			+ "	addr dc:a6:32:a6:e0:c2\n"
+        			+ "	ssid kura_gateway_raspberry_pi\n"
+        			+ "	type AP\n"
+        			+ "	wiphy 0\n"
+        			+ "	channel 1 (2412 MHz), width: 20 MHz, center1: 2412 MHz\n"
+        			+ "	txpower 31.00 dBm";
+        	
+        	String commandOutput3 = "Wiphy phy0\n"
```

Comment:
Create a few private static strings and give them some meaningful names

To make it easier here's a brief description:
- `commandOutput` is the output of the command `iw reg get` and is parsed for retrieving the country code.
- `commandOutput2' is the output of the command `iw wlan0 info` and is parsed for retrieving the physical device implementing the `wlan0` virtual device.
- `commandOutput3` is the output of the command `iw phy phy0 info` and is parsed for retrieving the supported channels by the wireless network interface.

The code responsible for doing this can be found here:

**Country code**

https://github.com/eclipse/kura/blob/1abec3bcef1bc72b422e6d8419bab89cc6faac44/kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/IwCapabilityTool.java#L227-L248

**Supported channels**

https://github.com/eclipse/kura/blob/1abec3bcef1bc72b422e6d8419bab89cc6faac44/kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/IwCapabilityTool.java#L173-L193